ote/src/cljs/ote/ui/link_icon.cljs

```diff
@@ -0,0 +1,16 @@
+(ns ote.ui.link-icon
+  (:require [stylefy.core :as stylefy]
+            [ote.style.base :as style-base]))
+
+
+(defn link-with-icon
+  [icon url link-text on-click]
+  [:div
+   [:a (merge {:href url
+               :style {:margin-right "2rem"}
+               :id "edit-transport-operator-btn"}
```

Comment:
We can't know where this button is used so the id can't be named as transport operator